jinja2/utils.py

```diff
@@ -512,6 +511,8 @@ def __next__(self):
         self.pos = (self.pos + 1) % len(self.items)
         return rv
 
+    next = __next__
```

Comment:
I'd do it the other way round and rename the method since a method named `next()` is part of the public API while `__next__` is just a nice extra.